### DISCUSSION TITLE
render emojis in the title top bar?

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#4458] [Q&A] [answered]
I wonder if it is technically possible to make emojis appear rendered when the heading title is displayed in the top bar?

https://user-images.githubusercontent.com/5679861/194075441-6ecfbe40-e6ac-4c7e-8e49-9d5d22781f3b.mp4

**Comment by squidfunk** [comment, marked as answer]
You could fall back to emoji code points, but the `:...:` syntax will not work, since the title is sourced from MkDocs title detection logic, which is based on four factors and doesn't go through Markdown, IIRC. It's on the MkDocs docs, but I can't find it right now.

**Reply by hellt** [reply]
Thanks. Yes, a unicode char is a nice fallback
For completeness, and for my future self

1. Go to https://unicode.org/emoji/charts/full-emoji-list.html
2. copy the emoji char from the table
3. paste in the editor
4. enjoy a tad different styling, but a working functionality for titles

<img width="1013" alt="image" src="https://user-images.githubusercontent.com/5679861/194232213-3b8fb5d7-f33f-4f8e-9185-edb22b29782e.png">
---
<img width="1153" alt="image" src="https://user-images.githubusercontent.com/5679861/194232258-e4010880-0147-4a31-9b08-33cb81ec3fd0.png">
